CMakeLists.txt

```diff
@@ -171,12 +171,15 @@ if(LINUX)
   set(HAVE_BLKID ${BLKID_FOUND})
   find_package(keyutils REQUIRED)
   set(HAVE_KEYUTILS ${KEYUTILS_FOUND})
+  find_package(libcap REQUIRED)
+  set(HAVE_LIBCAP ${LIBCAP_FOUND})
```

Comment:
if libcap is "REQUIRED", probably we can assume the existence of this library and hence do not need to check for`LIBCAP_FOUND`?